contrib/plugins/cowsay.sh

```diff
@@ -0,0 +1,25 @@
+#! /bin/sh
+
+# Eg. {"jsonrpc":"2.0","id":2,"method":"getmanifest","params":{}}\n\n
+read JSON
+read BLANK
+id=$(echo "$JSON" | sed 's/.*"id" *: *\([0-9]*\),.*/\1/')
+
+echo '{"jsonrpc":"2.0","id":'$id',"result":{"options":[],"rpcmethods":[{"name":"cowsay","usage":"<string>","description":"Have a cow, man!"}]}}'
```

Comment:
Consider replacing `$id` with `"$id"` to prevent potential globbing and word splitting. Applies to all echoing of `$id` in the script.